src/EventStore.ClientAPI/ConnectionSettings.cs

```diff
@@ -37,7 +37,7 @@ public sealed class ConnectionSettings {
 		/// <summary>
 		/// Allows overriding the HTTPClient <see cref="IHttpClient"/>
 		/// </summary>
-		public IHttpClient CustomHttpClient { get; set; }
+		public IHttpClient CustomHttpClient;
```

Comment:
What reason is there to change this from a property?

Reply:
`typeof(ConnectionSettings).GetFields(BindingFlags.Instance | BindingFlags.Public)` does not return the `CustomHttpClient` in the fields list here: https://github.com/EventStore/EventStore/pull/2553/files#diff-27fb9d7199f6a8cefc173c874380ed55L99

(It is required for connection strings)

Reply:
okay, not ideal, but less trouble than doing anything else i guess